### PR TITLE
Fix: Position of snackbar is too high

### LINKED ISSUE #708: Position of snackbar is too high
Context:
https://github.com/DroidKaigi/conference-app-2024/pull/675

### PR DESCRIPTION
**Pull request by garijp**
## Issue
- close #708 

## Overview (Required)
- Change GlassLikeBottomNavigation's padding.
- Adjust bottomPadding for each screen.

## Supplement
- When adjusting the padding of GlassLikeBottomNavigation, I noticed that the UI at the bottom of each screen was touching the Navigation, so I added about 16dp of padding.
- The event map and about screen were fine so I didn't adjust the padding.

## Screenshot (Optional if screenshot test is present or unrelated to UI)
Before | After
:--: | :--:
 |

### CODE REVIEW

File: feature/profilecard/src/commonMain/kotlin/io/github/droidkaigi/confsched/profilecard/ProfileCardScreen.kt [hunk 1]
```diff
@@ -206,7 +206,7 @@ internal fun ProfileCardScreen(
             left = contentPadding.calculateLeftPadding(layoutDirection),
             top = contentPadding.calculateTopPadding(),
             right = contentPadding.calculateRightPadding(layoutDirection),
-            bottom = contentPadding.calculateBottomPadding(),
+            bottom = contentPadding.calculateBottomPadding().plus(16.dp),
```
Comment (takahirom) [inline feature/profilecard/src/commonMain/kotlin/io/github/droidkaigi/confsched/profilecard/ProfileCardScreen.kt:209]: Thank you for finding the solution!
How about adding a comment like this?

```
// Adjusting Snackbar position
.plus(16.dp)
```

Comment (garijp) [reply feature/profilecard/src/commonMain/kotlin/io/github/droidkaigi/confsched/profilecard/ProfileCardScreen.kt]: Comment added.

**Comment by garijp** [comment]
Conflict resolved.

**Comment by garijp** [comment]
I'm glad you know about the app. Thank you. 😄

**Pull request merged**